src/Provider/TileProvider.js

```diff
@@ -80,8 +80,7 @@ function executeCommand(command) {
     };
 
     geometry._count++;
-    const tile = new TileMesh(geometry, params);
-    tile.layer = layer;
+    const tile = new TileMesh(layer, geometry, params);
```

Comment:
same question, you remember where `tile.layer` is used?

Reply:
Answered above.